config/Shared/config_default.php

```diff
@@ -147,6 +149,8 @@
     PaymentRefundFailedTransfer::class => 'payment-events',
     PaymentCanceledTransfer::class => 'payment-events',
     PaymentCancellationFailedTransfer::class => 'payment-events',
+    PaymentOverpaidTransfer::class => 'payment-events',
+    PaymentUnderpaidTransfer::class => 'payment-events',
```

Comment:
This requires to be added in events for MB1 for all envs, was it done already?
Because I cannot find 'em in configuration_app DynamoDB table